application/asset/js/resource-selector.js

```diff
@@ -52,8 +53,34 @@
         });
 
         $('#select-resource').on('click', '.quick-select-toggle', function() {
-            $(this).toggleClass('active');
+            $(this).toggleClass('active').find('button .sr-only').toggleClass('active');
```

Comment:
Little odd to toggle the "active" class on the screenreader text inside the button but not the "inactive" class... I don't quite get why this is necessary. You have a comment on the corresponding styles but I'm not getting the distinction.

Reply:
I don't think it's necessary after all. I think these classes and their toggling got away from me, gonna pare them down.